### PR TITLE
Add known Calibration parameter tag

### PR DESCRIPTION
**Pull request by peterbarker**
```
<parameters name="COMPASS_">
<param humanName="Compass offsets in milligauss on the X axis" name="COMPASS_OFS_X" documentation="Offset to be added to the compass x-axis values to compensate for metal in the frame" user="Advanced" calibration="1">
<field name="Calibration">1</field>
<field name="Range">-400 400</field>
<field name="Increment">1</field>
<field name="Units">mGauss</field>
<field name="UnitText">milligauss</field>
```

### CODE REVIEW

**Comment by amilcarlucas** [comment]
I can see why this is interesting. What uses are you planning for it ?

**Comment by peterbarker** [comment]
@amilcarlucas primarily it allows the GCS to filter out parameters the user is *very* unlikely to want to change.  It could have a checkbox for showing them or not, or avoid sending them to the vehicle on an upload without a force option.

File: libraries/AP_Compass/AP_Compass.cpp [hunk 1]
```diff
@@ -170,6 +178,8 @@ const AP_Param::GroupInfo Compass::var_info[] = {
     // @Units: mGauss
     // @Increment: 1
     // @User: Advanced
+    // @Calibration: 1
+    // @Calibration: 1
```
Comment (tridge) [inline libraries/AP_Compass/AP_Compass.cpp]: doubly calibrated?

**Comment by amilcarlucas** [comment]
Nice!!

**Comment by peterbarker** [comment]
On Mon, 3 Feb 2020, Andrew Tridgell wrote:

> +    // @Calibration: 1
> +    // @Calibration: 1
> 
> doubly calibrated?

Much more preciserer.

**Pull request merged**